test/parallel/test-http-client-response-timeout.js

```diff
@@ -9,6 +9,6 @@ server.listen(common.mustCall(() => {
     http.get({ port: server.address().port }, common.mustCall((res) => {
       res.on('timeout', common.mustCall(() => req.destroy()));
```

Comment:
```suggestion
      res.on('timeout', common.mustCall(() => {
        req.destroy();
        server.close();
      }));
```

The original issue was that the listener of the `'timeout'` event was not called if added on the `response` object so this should be ok. It does not invalidate the test and only one timer is used.

Reply:
Failure will require a timeout and kill from the runner in this case?

Reply:
> Failure will require a timeout and kill from the runner in this case?

Yes.